app/services/ui_service_mixin.rb

```diff
@@ -24,7 +24,7 @@ def icons
       :NetworkRouter           => {:type => "glyph", :class => 'pficon pficon-route'},
       :PhysicalRack            => {:type => "glyph", :class => 'pficon pficon-enterprise'},
       :PhysicalServer          => {:type => "glyph", :class => 'pficon pficon-server-group'},
-      :PhysicalSwitch          => {:type => "glyph", :class => 'pficon pficon-container-node'},
+      :PhysicalSwitch          => {:type => "glyph", :class => 'ff ff-network-switch'},
```

Comment:
Cc @saulotoledo (because the original fonticon for physical switch comes from https://github.com/ManageIQ/manageiq-ui-classic/pull/3699)

Reply:
I confirm that the change here is ok. I should have done this before. Thanks @himdel for the note, and @douglasgabriel for the changes.